cmake/apply_lvi_mitigation.cmake

```diff
@@ -18,45 +18,51 @@ endmacro ()
 function (apply_lvi_mitigation NAME)
   # Add LVI mitigation compliation options.
   if (UNIX)
-    if (CMAKE_C_COMPILER_ID MATCHES Clang)
-      # Enforce clang to invoke the gnu assembler instead of the integrated one.
+    if (LVI_MITIGATION STREQUAL ControlFlow-Clang)
+      # Enable clang-11 built-in LVI mitigation
+      target_compile_options(${NAME} PRIVATE -mlvi-cfi)
```

Comment:
@asvrada  - what was the perf difference between CF-Clang and CF-Gnu

Reply:
I only run performance tests between no LVI and Clang LVI. Will try to get the result for GNU if possible